registry/config.go

```diff
@@ -2,4 +2,7 @@ package registry
 
 var (
 	defaultV2Registry = "registry-1.docker.io"
+
+	// DefaultRegistry set default registry value
+	DefaultRegistry = "https://index.docker.io/v1/"
```

Comment:
I find another default registry in `mgr.config.DefaultRegistry`, do we need to define a new one here?